src/include/distributed/transaction_identifier.h

```diff
@@ -21,6 +21,10 @@
  *
  *  -  initiatorNodeIdentifier: A unique identifier of the node that initiated
  *     the distributed transaction
+ *  -  transcationStartedOnNode: Set to true only for the transactions initialized on
```

Comment:
Maybe rename to transactionOriginator or transactionMaster or somesuch? Because StartedOnNode could as well be taken to be true for all mx queries reaching the local node...